### PR TITLE
endpoint enrichment generic playbook

### PR DESCRIPTION
**Pull request by erezh31**
fixes: https://github.com/demisto/etc/issues/8042

### CODE REVIEW

File: Playbooks/playbook-Endpoint_Enrichment_-_Generic.yml [hunk 1]
```diff
@@ -0,0 +1,901 @@
...
+    simple: ${Endpoint.Hostname}
+  required: false
+  description: The hostname to enrich
+outputs:
+- contextPath: Endpoint.Hostname
+  description: Endpoint hostname
+  type: string
+- contextPath: Endpoint.Mac
+  description: Endpoint mac addresses
+  type: unknown
```
[the remark this answers is not in the document]
Comment (erezh31) [reply Playbooks/playbook-Endpoint_Enrichment_-_Generic.yml]: it's a list - there is no such type

File: Playbooks/playbook-Endpoint_Enrichment_-_Generic.yml [hunk 2]
```diff
@@ -0,0 +1,901 @@
...
+- contextPath: Endpoint.OS
+  description: Endpoint OS
+  type: string
+- contextPath: Endpoint.IP
+  description: Endpoint ip addresses
+  type: unknown
```
[the remark this answers is not in the document]
Comment (erezh31) [reply Playbooks/playbook-Endpoint_Enrichment_-_Generic.yml]: it's a list - there is no such type

File: Playbooks/playbook-Endpoint_Enrichment_-_Generic.yml [hunk 3]
```diff
@@ -0,0 +1,901 @@
...
+      value:
+        complex:
+          root: modules
+          filters:
+          - - operator: string.isEqual
+              left:
+                value: brand
+                iscontext: true
+              right:
+                value: active
```
[the remark this answers is not in the document]
Comment (erezh31) [reply Playbooks/playbook-Endpoint_Enrichment_-_Generic.yml]: fixed

**Comment by meirwah** [comment]
Schema validation errors, @erezh31 plz fix

**Comment by asafshen** [comment]
@erezh31 
we have made some changes in playbook structure - please download the playbook again from an updated demisto env (demo2/your local env)

**Comment by erezh31** [comment]
@itaimajlin @anara123 please review again - ci passed

**Pull request merged**